package.json

```diff
@@ -5,9 +5,12 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "t": "ts-node"
   },
-  "dependencies": {},
+  "dependencies": {
+    "mermaid": "^8.14.0"
```

Comment:
it's all dev packages actually as in prod mode this stuff is not used to generate docs, as of now :)

Reply:
ok